ddev/src/ddev/integration/core.py

```diff
@@ -139,6 +139,13 @@ def minimum_base_package_version(self) -> str | None:
 
         return None
 
+    @cached_property
```

Comment:
Is it a good idea to cache it since some commands update this file?

Reply:
Good point. We could probably implement a way to invalidate the cache but I don't think it's worth it, so I'm turning it into a regular `property` to be safe.